packages/blast-utilities/contract-utils.js

```diff
@@ -27,7 +27,8 @@ module.exports.CudosContract = class CudosContract {
     }
   }
 
-  async deploy(initMsg, funds, label = this.#contractLabel) {
+  async deploy(initMsg, signer = this.#signer, label = this.#contractLabel, funds) {
+    this.#signer = signer
```

Comment:
I think that it will be better to use signer, label and fund params as an optional object params

Reply:
Using object that wraps signer, label and fund instead of directly signer, label and fund as parameters will introduce breaking change. This task aims to remove breaking changes. Your suggestion is included in another task.